website/docs/opengl/c1/3-hello-texture.md

```diff
@@ -0,0 +1,388 @@
+```c#
+_gl.ActiveTexture(TextureUnit.Texture0);
+```
+And after, bind the texture!
+```c#
+_gl.BindTexture(TextureTarget.Texture2D, _texture);
+```
+
+And now when you run it (drumroll...), you can see the image being drawn inside the quad!
+
+![Quad with texture](../../../images/opengl/chapter1/leasson3/quad-with-texture.png)
+
+## Texture parameters
+And now, some extra content to go with this tutorial!
+
+If you get curious about how the texture parameters work, let's learn it now!
+
+First, let's learn the structure of the command:
+```c#
+_gl.TextureParameter( [Texture ID] , [Parameter to change] , [Value to change] );
+```
+
+The texture ID you already know. So let's learn what each parameter means!
+
+### `TextureWrapS` & `TextureWrapT`:
+`TextureWrap` is the parameter that allows you to say what the `texture` method (in the fragment shader) should do with values less than 0 and
+greater than 1. The `S` and `T` refer to X and Y respectively.
+
+Let's see the most common values for these parameters:
+
+#### `TextureWrapMode.Repeat`:
+Just repeats the image without any change, like the value never got less than 0 or greater than 1.  
+![Repeat](../../../images/opengl/chapter1/leasson3/texParameters/repeat.png)
+
+#### `TextureWrapMode.MirroredRepeat`:
+Mirror the texture for each 1 texture coordinate unit.  
+![Mirrored](../../../images/opengl/chapter1/leasson3/texParameters/repeatMirrored.png)
+
+#### `TextureWrapMode.ClampToBorder`:
+Returns a solid color when the coordinate is less than 0 or greater than 1.  
+![A big border](../../../images/opengl/chapter1/leasson3/texParameters/clampToBorder.png)
+
+#### `TextureWrapMode.ClampToEdge`:
+Returns the pixel on the respective edge of the image.  
+![A weird border](../../../images/opengl/chapter1/leasson3/texParameters/clampToEdge.png)
+
+### `TextureMinFilter` & `TextureMagFilter`:
+Texture `min` and `mag` filters are the filters used when the texture's final size is, respectively, less than or greater than the original size.
+For now, we will show just the main two options and in the next section (Mipmaps), some others will be shown.
+
+
+#### `Texture(Min/Mag)Filter.Linear`:
+The (bi)linear filter is the best for low-quality images. When the pixels are sampled from the texture, this filter will get the color of the nearest
+pixels of the pixel in the center of the texture coordinates and will return a linear interpolation of them.
+
+This is an example from [Learn OpenGL](https://learnopengl.com/Getting-started/Textures). See how the neighbor colors are interpolated to return a different
+color:
+
+![linear filter](../../../images/opengl/chapter1/leasson3/texParameters/filter_linear.png)
+
+
+#### `Texture(Min/Mag)Filter.Nearest`:
+The nearest filter returns the color of the center of the nearest pixel, no interpolation is done.
+
+![Nearest filter](../../../images/opengl/chapter1/leasson3/texParameters/filter_nearest.png)
+
+## Mipmaps
```

Comment:
iirc mipmapping is not part of the original tutorial checked into the repo. I'm not objecting to searching mipmapping, but it is important that the examples folder and the web tutorials stay in sync

Reply:
The tutorial source uses mipmap generation call and there's no other place to explain about it. This is an extra section so i don't see any problem

Reply:
Thanks for confirming